src/client/components/common/Card/CardHeader/index.tsx

```diff
@@ -1,9 +1,10 @@
 import { FC } from 'react';
-import styled from 'styled-components';
+import styled from 'styled-components/macro';
 
 const Container = styled.div`
   width: 100%;
   padding: 0 ${({ theme }) => theme.card.padding};
+  margin-bottom: 1.2rem;
```

Comment:
Same here, We can have a cardHeader without content below, and this can create a weird space below it